clients/java-logger/library/src/test/java/com/abcxyz/lumberjack/auditlogclient/config/BackendContextTest.java

```diff
@@ -19,17 +19,26 @@
 import static org.assertj.core.api.Assertions.assertThat;
 
 import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
 import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
 import java.io.IOException;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
 import org.mockito.junit.jupiter.MockitoExtension;
 
 @ExtendWith(MockitoExtension.class)
 public class BackendContextTest {
+  @BeforeEach
+  void init() {
+    ObjectMapper mapper = new ObjectMapper(new YAMLFactory());
```

Comment:
Did you run tests locally? It should require mapper to be a member variable in this class for tests in this class to access it. Also, to align with the rest, use `public void init()`
```
private ObjectMapper mapper;
```

Reply:
Done